hub-backend/src/main/java/org/panda_lang/hub/authentication/OAuth2AuthorizationRequestRepository.kt

```diff
@@ -22,18 +22,19 @@ import javax.servlet.http.Cookie
 import javax.servlet.http.HttpServletRequest
 import javax.servlet.http.HttpServletResponse
 
+private const val COOKIE_EXPIRATION_TIME = 180
+private const val OAUTH_2_AUTHORIZATION_REQUEST_COOKIE_NAME = "oauth2_auth_request"
+
 internal class OAuth2AuthorizationRequestRepository : AuthorizationRequestRepository<OAuth2AuthorizationRequest?> {
 
     companion object {
         const val REDIRECT_URI_COOKIE_NAME = "redirect_uri"
-        private const val COOKIE_EXPIRATION_TIME = 180
-        private const val OAUTH_2_AUTHORIZATION_REQUEST_COOKIE_NAME = "oauth2_auth_request"
```

Comment:
These constant values are specific to this class and because this class is the only item in the file, we may lift them out to the top level. The non-private const val stays as other pieces of code depends on it.

This has been done for the other classes following this pattern and in return removed some now unused companion objects.